docs/sctp.md

```diff
@@ -0,0 +1,152 @@
+
+> **Note**: SCTP protocol support is not available for service type NodePortLocal, as Antrea CNI does not support SCTP Service ports, for NodePortLocal type services.
+
+## Configuration
+
+As already stated, the configuration mainly includes creating LoadBalancer services, and Gateway objects and their corresponding backend services, with SCTP protocol in port definitions. Some sample yaml definitions are shown below.
+
+### LoadBalancer Service
+
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sctp-demo
+  labels:
+    app: server
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: server
+  template:
+    metadata:
+      labels:
+        app: server
+    spec:
+      containers:
+      - name: sctp-demo
+        image: <sctp enabled container image>
+        ports:
+        - containerPort: 9090
+          protocol: SCTP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: server
+  namespace: default
+spec:
+  ports:
+  - port: 80
+    protocol: SCTP
+    targetPort: 9090
+  selector:
+    app: server
+  type: LoadBalancer
+```
+
+### Gateway Object
+
+```
+apiVersion: networking.x-k8s.io/v1alpha1
+kind: GatewayClass
+metadata:
+  name: avi-lb
+spec:
+  controller: ako.vmware.com/avi-lb
+  parametersRef:
+    group: ako.vmware.com
+    kind: AviInfraSetting
+    name: my-infrasetting
+---
+apiVersion: ako.vmware.com/v1alpha1
+kind: AviInfraSetting
+metadata:
+  name: my-infrasetting
+---
+apiVersion: networking.x-k8s.io/v1alpha1
+kind: Gateway
+metadata:
+  name: my-gateway
+  namespace: svcapi
+spec:
+  gatewayClassName: avi-lb
+  listeners:
+  - port: 6060
+    protocol: SCTP
+    routes:
+      group: v1
+      kind: services
+      selector:
+        matchLabels:
+          ako.vmware.com/gateway-name: my-gateway
+          ako.vmware.com/gateway-namespace: svcapi
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    ako.vmware.com/gateway-name: my-gateway
+    ako.vmware.com/gateway-namespace: svcapi
+  name: avisvc-svcapi
+  namespace: svcapi
+spec:
+  ports:
+  - name: sixtysixty
+    port: 6060
+    protocol: SCTP
+    targetPort: 9090
+  selector:
+    app: avi-server
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: avi-server
+  namespace: svcapi
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: avi-server
+  template:
+    metadata:
+      labels:
+        app: avi-server
+    spec:
+      containers:
+      - image: <sctp enabled container image>
+        imagePullPolicy: IfNotPresent
+        name: avi-server
+        ports:
+        - containerPort: 9090
+          protocol: SCTP
+```
+> **Note**: The above example for Gateway defines service of type ClusterIP as the backend service. But, service of type NodePort can also be used.
+
+### FAQ
```

Comment:
We have to move this faq to this faq doc https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/master/docs/faq.md

Reply:
Moved.